homeassistant/components/microsoft_face_detect/image_processing.py

```diff
@@ -112,14 +110,14 @@ async def async_process_image(self, image):
         if not face_data:
             face_data = []
 
-        faces = []
+        faces: list[FaceInformation] = []
         for face in face_data:
             face_attr = {}
             for attr in self._attributes:
                 if attr in face["faceAttributes"]:
                     face_attr[attr] = face["faceAttributes"][attr]
 
             if face_attr:
-                faces.append(face_attr)
+                faces.append(face_attr)  # type: ignore[arg-type]
```

Comment:
why the ignore?

Reply:
Because faces expects objects of type `FaceInformation(TypedDict)`

I've adjusted the code to narrow down the type ignore with `literal-required`